pkg/controller/che/che_controller.go

```diff
@@ -236,7 +231,7 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
-	if isOpenShift {
+	if util.IsOpenShift {
```

Comment:
I missed this previous time and maybe that is not for current PR, but getting a context value from `util` look pretty weird. To me, it should be in some sort of context.

Reply:
Sure, It is in my todo list.